scripts/actions/check-job-progress.js

```diff
@@ -160,7 +175,10 @@ const updateTranslationRecords = async (slugStatuses) => {
 
   await Promise.all(
     slugStatuses.map(async ({ ok, slug }) => {
-      const updateStatus = ok ? StatusEnum.COMPLETED : StatusEnum.ERRORED;
+      if (!ok) {
+        return;
+      }
+      const updateStatus = StatusEnum.COMPLETED;
```

Comment:
this is probably the only change we need -- it handles removing the errored state from being set. additionally, if everything is set to completed, all records will be deleted at the end.

we can still log the failed slugs, that seems prudent.